working/augmentation-libraries/feature-specification.md

```diff
@@ -924,13 +924,29 @@ It is a compile-time error if:
 When augmenting an extension type declaration, the parenthesized clause where
 the representation type is specified is treated as a constructor that has a
 single positional parameter, a single initializer from the parameter to the
-representation field, and an empty body.
-
-This means that an augmentation can add a body to an extension type's
-constructor, which isn't otherwise possible. *(But note that there is no
+representation field, and an empty body. This clause must be present on the
+declaration which introduces the extension type, and must be ommitted from all
```

Comment:
Spelling: "ommitted" -> "omitted".

Also, "this clause" is ambiguous since the previous sentence lists a bunch of stuff. How about:

"This clause" -> "The representation field clause".